proxy/src/main/java/net/md_5/bungee/util/ChatComponentTransformer.java

```diff
@@ -63,7 +63,7 @@ public static ChatComponentTransformer getInstance()
         {
             if ( root.getExtra() != null && !root.getExtra().isEmpty() )
             {
-                List<BaseComponent> list = Lists.newArrayList( transform( player, root.getExtra().toArray( new BaseComponent[ root.getExtra().size() ] ) ) );
+                List<BaseComponent> list = Arrays.asList( transform( player, root.getExtra().toArray( new BaseComponent[root.getExtra().size()] ) ) );
```

Comment:
If you're going to do this you should do ```new ArrayList<>(Arrays.asList(...));```

Reply:
Sure. Will change tomorrow